consensus/enclave/impl/build.rs

```diff
@@ -33,15 +33,19 @@ fn main() {
 
     rerun_if_env_changed!("FEE_SPEND_PUBLIC_KEY");
     rerun_if_env_changed!("FEE_VIEW_PUBLIC_KEY");
+    rerun_if_env_changed!("GOVERNOR_ADMIN_PUBLIC_KEY");
```

Comment:
No, because the actual master minters are being renamed to governors, and this key "sits on top" of them. Governors governing key is confusing, hence we decided to go with governors admin key. But the change from master minters to governors hasn't happened yet.

Reply:
Do we really want to call it illuminatus key? I have no strong feeling about how to name this thing.

Reply:
I don't have any dogs in any fights, but it sounded like there was confusion in this PR discussion around "governors vs governors admin," so we should pick a name that's easier to should be very easy to tell which key is the root of trust.

Reply:
`MINTING_TRUST_ROOT`?